src/hypomnema/xml/backends/__init__.py

```
@@ -14,15 +14,10 @@
-# via a subprocess in test_imports.py
+except ImportError as e:
+  from warnings import warn
 
-except ImportError as e:  # pragma: no cover
-  from warnings import warn  # pragma: no cover
-
-  warn(
-    f"lxml not installed, Only StandardBackend will be available. Error: {e}"
-  )  # pragma: no cover
-  LxmlBackend = None  # type:ignore # pragma: no cover
+  warn(f"lxml not installed, Only StandardBackend will be available. Error: {e}")
```

Ruff (0.15.1)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)